pyarrow.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: pyarrow
+  epoch: 0
+  version: 14.0.0
+  description: "Apache Arrow Python bindings"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - apache-arrow-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - cython
+      - numpy
+      - openssl-dev
+      - py3-pip
+      - py3-setuptools
+      - python3
+      - python3-dev
+
+pipeline:
+  - uses: git-checkout
+    working-directory: /home/build/apache-arrow
+    with:
+      repository: https://github.com/apache/arrow.git
```

Comment:
Should this be a subpackage of arrow.yaml so they stay in sync?

Reply:
Ideally yes, I tried all day to be able to build it under there but couldn't find a way to make it build under there because of cmake not finding all the stuff.